examples/scripts/SPMe_step.py

```diff
@@ -32,12 +32,13 @@
 
 # step model
 dt = 500
+t_eval = np.array([0, 50, 100, 200, 500])
```

Comment:
can you add a comment that t_eval are the times in the interval [0, dt]?

Reply:
and that `dt` is the size of the step (I think it's pretty clear but let's be explicit)